### ISSUE TITLE
Web interface needs to be separated out from JSON API

### ISSUE THREAD

**Issue by Japh** [homedash/kankun-json#10]
At the moment, we have two projects in one repository, which is kind of unwieldy.
1. The web interface
2. The JSON.cgi API

There's no reason to install the web interface on every device when it can control them all from a single location (on your desktop even), as long as you have the JSON.cgi API installed on the devices.

We should really separate these out, both to save space on the devices, but also to simplify development.

**Comment by Japh** [comment]
I've created a new repo to be the home for this: [homedash/homedash-client](https://github.com/homedash/homedash-client).

Feel free to open relevant issues over there.

**Comment by mattjoyce** [comment]
Yep, separate is the way to go.  Easier to play with front end ideas too.

**Comment by stokito** [comment]
We can simplify by using [OpenWrt ubus](https://openwrt.org/docs/techref/ubus)
jshn can be used to generate JSON.
TBH the UI is anyway small and if anybody don't want UI the can just install json.sh
